docs/detections/prebuilt-rules/downloadable-packages/0-13-1/prebuilt-rules-0-13-1-summary.asciidoc

```diff
@@ -0,0 +1,150 @@
+[[prebuilt-rule-0-13-1-prebuilt-rules-0-13-1-summary]]
+[role="xpack"]
+== Prebuilt rules updates for downloadable package 0.13.1
```

Comment:
```suggestion
== Prebuilt Security Detection Rules update v0.13.0
```
also, I noticed that this isn't rendering in the TOC properly. ideas?
<img width="385" alt="Screen Shot 2021-06-22 at 12 35 02 PM" src="https://user-images.githubusercontent.com/31489089/122980762-4698cf80-d356-11eb-8b37-5d3548ad2f28.png">